test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-1.xml

```diff
@@ -1058,9 +1058,9 @@
 	
 	<test id="Test 57-a: Make sure that lambda classes work and get stored in the shared class cache" timeout="600" runPath=".">
 		<command>$JAVA_EXE$ $currentMode$ -Xtrace:print={j9shr.2259} $CP_HANOI$ $PROGRAM_LAMBDA$ 0</command>
-		<output type="success" caseSensitive="no" regex="yes" javaUtilPattern="yes">j9shr[\.]2259     > API j9shr_classStoreTransaction_createSharedClass : enter [\(]classname=org/openj9/test/lambdatests/Test1..Lambda.([\d]+)/(00000000|0000000000000000) </output>
+		<output type="success" caseSensitive="no" regex="yes" javaUtilPattern="yes">j9shr[\.]2259     > API j9shr_classStoreTransaction_createSharedClass : enter [\(]classname=org/openj9/test/lambdatests/Test1..Lambda.([\d]+)/(0000000000|000000000000000000) </output>
```

Comment:
Why are we checking for 2 more `0` ? Shouldn't we be checking for `0x` ?

Reply:
The `0x` is only written after we have the romAddress, the tracepoint that this test relies on is called before then

Reply:
0x is written along with the romAddress

Reply:
OK. Thanks.

Reply:
Probably better to make a comment in this file explaining why 2 more '0' are been checked.

Reply:
ill add a comment once the tests have passed